datalad/support/configparserinc.py

```diff
@@ -17,36 +17,36 @@
 
 if sys.version_info >= (3,2): # pragma: no cover
 
-	# SafeConfigParser deprecated from Python 3.2 (renamed to ConfigParser)
```

Comment:
Now will become hard to pick up fixes from fail2ban's copy :-(
Was it screaming out loud about tabs? But OK let's do it for uniformity